lib/teiserver/coordinator/consul_server.ex

```diff
@@ -255,6 +255,12 @@ defmodule Teiserver.Coordinator.ConsulServer do
         Coordinator.cast_coordinator({:consul_command, Map.merge(cmd, %{lobby_id: state.lobby_id, host_id: state.host_id})})
         {:noreply, state}
 
+      Enum.member?(@always_allow, cmd.command) == false and
+      Enum.member?(@boss_commands, cmd.command) == false and
+      Enum.member?(@host_commands, cmd.command) == false ->
+        LobbyChat.sayprivateex(state.coordinator_id, cmd.senderid, "No command of name '#{cmd.command}'", state.lobby_id)
+        {:noreply, state}
```

Comment:
Moderator commands are not part of these lists, this means every time a moderator command is called it will be caught by this block instead of the allowed check.

Reply:
Fixed